sqlconnect/internal/databricks/integration_test.go

```diff
@@ -49,6 +49,31 @@ func TestDatabricksDB(t *testing.T) {
 		)
 	})
 
+	t.Run("with oauth", func(t *testing.T) {
+		oauthConfigJSON, ok := os.LookupEnv("DATABRICKS_OAUTH_TEST_ENVIRONMENT_CREDENTIALS")
+		if !ok {
+			if os.Getenv("FORCE_RUN_INTEGRATION_TESTS") == "true" {
+				t.Fatal("DATABRICKS_OAUTH_TEST_ENVIRONMENT_CREDENTIALS environment variable not set")
+			}
+			t.Skip("skipping databricks integration test due to lack of a test environment")
+		}
+		configJSON, err := sjson.Set(oauthConfigJSON, "useOauth", true)
+		require.NoError(t, err, "failed to set useOauth")
+		_, err = sqlconnect.NewDB(databricks.DatabaseType, []byte(configJSON))
+		require.NoError(t, err, "failed to create db")
+
+		integrationtest.TestDatabaseScenarios(
+			t,
+			databricks.DatabaseType,
+			[]byte(configJSON),
+			strings.ToLower,
+			integrationtest.Options{
+				LegacySupport:                  true,
+				SpecialCharactersInQuotedTable: "_A",
```

Comment:
What's this use for? It's different from what is being used for `with information schema` suite.
```suggestion
				SpecialCharactersInQuotedTable: "_A",
```

Reply:
Not sure. I copied it as it is from an existing [test](https://github.com/rudderlabs/sqlconnect-go/blob/1ba5420ad9695c0d18bac12524ae6ddc7fb34c7a/sqlconnect/internal/databricks/integration_test.go#L68)